packages/palette/src/elements/Banner/Banner.tsx

```diff
@@ -1,6 +1,6 @@
 import React from "react"
 import styled from "styled-components"
-import { CloseIcon } from "../../svgs"
+import { CloseIcon } from "../../svgs/CloseIcon"
```

Comment:
This is a side issue that was causing the docs build to fail. I'm not certain exactly why (because the close icon is imported to the index), but pointing it to the exact file fixed the error. Probably a best practice to do that anyway since those index files don't allow for tree shaking.

Reply:
yah, when we try to import from a bulk export list we get into circular dep problems